server/src/serializers/ReviewsSerializer.js

```diff
@@ -0,0 +1,30 @@
+import VotesSerializer from "./VotesSerializer.js"
+
+class ReviewsSerializer {
+  static getSummary(review) {
+    const allowedAttributes = ['id', 'title', 'body', 'rating', 'recipe']
+    let serializedReview = {}
+    for (const attribute of allowedAttributes) {
+      serializedReview[attribute] = review[attribute]
+    }
+    return serializedReview
+  }
+
+  static async getDetails(review) {
+    const serializedReview = this.getSummary(review)
+    const relatedVotes = await review.$relatedQuery("votes")
+    const serializedVotes = relatedVotes.map((vote) => {
+      return VotesSerializer.getSummary(vote)
+    })
+    serializedReview.votes = serializedVotes
+    serializedReview.voteCount = await VotesSerializer.getVoteCount(review.id)
+    let usersVoted = await VotesSerializer.getReviewers(review.id)
```

Comment:
VoteSerializer.getSummary should serialize the Vote AND get the VoteCount AND get the reviewers. All in one method.